io.github.atari800.Atari800.yml

```diff
@@ -0,0 +1,54 @@
+app-id: io.github.atari800.Atari800
+runtime: org.kde.Platform
+runtime-version: '6.4'
+sdk: org.kde.Sdk
+command: atari800
+build-options:
+  no-debuginfo: true
+  strip: true
```

Comment:
```suggestion
```

strip is default. and debuginfo is extracted.

Reply:
Removed.